### ISSUE TITLE
MacOS Support

### ISSUE THREAD

**Issue by alexmozzhakov** [SigPloiter/SigPloit#101]
When running an ./installer.sh installation gets stopped by sed error
```
sed: illegal option -- u
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```
A problem command is `sed -u -e "s,\.,,g" `

**Comment by alexmozzhakov** [comment]
Also, getting java.io.FileNotFoundException when running testing server jars. Since no source provided, I have no opportunity to fix folder to local, and not 
`/home/gh0/restcomm-jss7-7.0.1383/ss7/maplog.log`

**Comment by SigPloiter** [comment]
Alexey, skip the installer.sh it's not needed I will remove it

**Comment by SigPloiter** [comment]
I will upload the src codes in a while no problem

**Comment by alexmozzhakov** [comment]
Actually, I would like to also remove installation_readme and installation_requirements and put run command and Java/Python requirements in README

**Comment by SigPloiter** [comment]
@alexmozzhakov source codes uploaded in their corresponding server side paths

**Comment by alexmozzhakov** [comment]
Seems like log4j is not properly set up, or I am missing some config files. After I've tried to build server on my own machine I got 
```
log4j:WARN No appenders could be found for logger (org.mobicents.protocols.sctp.ManagementImpl).
log4j:WARN Please initialize the log4j system properly.
```
on both 1.x and 2.x versions of log4j. But they're just warnings, not errors. 
No traces of hard coded local path found, so I guess, that built jars are just outdated, so they produce FileNotFoundException. Since I've no experience dealing with log4j, I would appreciate if you will fix this problem yourself, rebuilding jars and configuring logger.
